src/include/roctx.h

```diff
@@ -11,7 +11,9 @@
 #include <string.h>
 #include <map>
 
+#ifndef ROCTX_NO_IMPL
```

Comment:
if roctracer-is not installed on the systemdev, and one still uses `--roctx-enable` with `./install.sh`, we get make errors like:
```
In file included from /home/amd/nnegi/rccl_wenkaidu_roctx/build/release/hipify/src/misc/roctx.cc:7:
/home/amd/nnegi/rccl_wenkaidu_roctx/build/release/hipify/src/include/roctx.h:15:10: fatal error: 'roctracer/roctx.h' file not found
#include <roctracer/roctx.h>
```
is this okay? or should there be a cmake error?

Reply:
Certainly it is better to have cmake warning on this. However, this is a different issue from what this PR attempts to fix.